createRecipeDb.sql

```diff
@@ -0,0 +1,331 @@
+    FOREIGN KEY (`recipe_id`) REFERENCES `Recipe` (`id`),
+    FOREIGN KEY (`step_id`) REFERENCES `Step` (`id`)
+);
+
+--
+-- Populate `RecipeStep` table with initial values
+--
+INSERT INTO `RecipeStep` VALUES
+    (1, 1, 1),
+    (2, 1, 2),
+    (3, 1, 3),
+    (4, 2, 4),
+    (5, 2, 5),
+    (6, 3, 6),
+    (7, 3, 7),
+    (8, 3, 8),
+    (9, 4, 9),
+    (10, 4, 10),
+    (11, 4, 11),
+    (12, 4, 12),
+    (13, 4, 13),
+    (14, 4, 14),
+    (15, 5, 15),
+    (16, 5, 16),
+    (17, 6, 17),
+    (18, 6, 18),
+    (19, 6, 19),
+    (20, 6, 20),
+    (21, 7, 21),
+    (22, 7, 22),
+    (23, 7, 23),
+    (24, 7, 24),
+    (25, 7, 25),
+    (26, 8, 26),
+    (27, 8, 27),
+    (28, 8, 28),
+    (29, 9, 29),
+    (30, 9, 30),
+    (31, 9, 31),
+    (32, 9, 32),
+    (33, 9, 33),
+    (34, 10, 34),
+    (35, 10, 35),
+    (36, 10, 36),
+    (37, 10, 37),
+    (38, 10, 38);
+
+-- ------------------------------------------------------------------
+
+--
+-- Table structure for table `User`
+--
+CREATE TABLE `User` (
+    `id`        INT             NOT NULL,
+    `username`  VARCHAR(30)     NOT NULL,
+    `password`  VARCHAR(30)     NOT NULL,
+    `logged_in` CHAR(1)         NOT NULL,
+    PRIMARY KEY (`id`)
+);
+
+--
+-- Populate `User` table with initial values
+--
+INSERT INTO `User` VALUES
+    (1, 'MuffinMan', 'muffinsrcool98', 'N'),
+    (2, 'idkwhatimdoing', 'sendhalp', 'N'),
+    (3, 'BakingFor83', 'whydoihavesomanykids', 'N'),
+    (4, 'CookingScrub97', 'ionlyeatramen', 'N'),
+    (5, 'CatsRCool', 'woof', 'N'),
+    (6, 'DogsRRad', 'meow', 'N'),
+    (7, 'healthyboi', 'ionlyeatkale', 'N'),
+    (8, 'wowza', 'whoa', 'N'),
+    (9, 'WhatIsAKitchen', 'ihavenevercookedinmylife', 'N'),
+    (10, 'Cooking4LYF', 'ihaveliterallyneverseentheoutsideofmykitchen', 'N');
+
+-- ------------------------------------------------------------------
+
+--
+-- Table structure for table `UserRecipe`
+--
+CREATE TABLE `UserRecipe` (
+    `id`            INT     NOT NULL,
+    `user_id`       INT     NOT NULL,
+    `recipe_id`     INT     NOT NULL,
+    PRIMARY KEY (`id`),
+    FOREIGN KEY (`user_id`) REFERENCES `User` (`id`),
+    FOREIGN KEY (`recipe_id`) REFERENCES `Recipe` (`id`)
+);
+
+--
+-- Populate `UserRecipe` table with initial values
+--
+INSERT INTO `UserRecipe` VALUES
+    (1, 1, 1),
+    (2, 2, 2),
+    (3, 3, 3),
+    (4, 4, 4),
+    (5, 5, 5),
+    (6, 5, 6),
+    (7, 5, 7),
+    (8, 8, 8),
+    (9, 9, 9),
+    (10, 10, 10);
+
+-- ------------------------------------------------------------------
+
+--
+-- Table structure for table `UserFavorite`
+--
+CREATE TABLE `UserFavorite` (
+    `id`            INT     NOT NULL,
+    `user_id`       INT     NOT NULL,
+    `recipe_id`     INT     NOT NULL,
+    PRIMARY KEY (`id`),
+    FOREIGN KEY (`user_id`) REFERENCES `User` (`id`),
+    FOREIGN KEY (`recipe_id`) REFERENCES `Recipe` (`id`)
+);
+
+--
+-- Populate `UserFavorite` table with initial values
+--
+INSERT INTO `UserFavorite` VALUES
```

Comment:
This bit doesn't work for me when bulk executing the SQL, but works when executing individually. Might just be doing something wrong on my end.

Reply:
I'll take a look at it and try replicating the issue.